scripts/grammar-out.txt

```diff
@@ -4,16 +4,19 @@
 ./demo/simple/reljam-xsd.sysl success
 ./demo/simple/sysl-sd.sysl success
 ./demo/simple/example2.sysl success
+./demo/simple/sysl-app-hyperlink.sysl success
```

Comment:
As a separate PR, can we move outcome to the front and first sort by that?

Reply:
I can try, I'll make a separate PR